mappings/net/minecraft/world/PersistentStateManager.mapping

```diff
@@ -8,10 +8,20 @@ CLASS net/minecraft/class_26 net/minecraft/world/PersistentStateManager
 		ARG 2 dataFixer
 	METHOD method_120 get (Ljava/util/function/Supplier;Ljava/lang/String;)Lnet/minecraft/class_18;
 		ARG 1 factory
-		ARG 2 id
+		ARG 2 fileName
 	METHOD method_123 set (Lnet/minecraft/class_18;)V
+		ARG 1 persistentState
 	METHOD method_125 save ()V
+	METHOD method_17921 isCompressedNbt (Ljava/io/PushbackInputStream;)Z
+		ARG 1 pushbackInputStream
 	METHOD method_17922 getFile (Ljava/lang/String;)Ljava/io/File;
-		ARG 1 id
+		ARG 1 fileName
+	METHOD method_17923 read (Ljava/lang/String;I)Lnet/minecraft/class_2487;
+		ARG 1 fileName
+		ARG 2 worldVersion
 	METHOD method_17924 getOrCreate (Ljava/util/function/Supplier;Ljava/lang/String;)Lnet/minecraft/class_18;
 		ARG 1 factory
+		ARG 2 fileName
+	METHOD method_20786 putIfAbsent (Ljava/util/function/Supplier;Ljava/lang/String;)Lnet/minecraft/class_18;
```

Comment:
This method attempts to retrieve the existing state from memory, and attempts to load it from disk if it fails. I would suggest `get`, given it is effectively the first part of the `getOrCreate` method.

Reply:
there is a method already named get, 
and yes it does what you said + it places the state retrieved from the disk on the loadedStates map, if it can't get it from the disk, it places null instead

Reply:
> there is a method already named get
Hence renaming the aforementioned `get` method. It works exactly like the Map contract: `getOrCreate` creates an object from scratch if `get` cannot obtain an object through normal means. The fact that the object is cached is an implementation detail IMO.

Reply:
https://github.com/FabricMC/yarn/pull/891/commits/e84a8b130c3161022e8941e12ce77883b50cc3bc